### ISSUE TITLE
qs add does not handle failure/multiple adds

### ISSUE THREAD

**Issue by maxandersen** [quarkusio/quarkus#16044]
## Describe the bug
`qs add metrics` fails (as expected as metadata atm is resulting in ambigous conflicts) with:

```
 qs add metrics
[FAILURE] ❌ Multiple extensions matching 'metrics'
     * io.quarkus:quarkus-smallrye-metrics
     * io.quarkus:quarkus-micrometer
     Be more specific e.g using the exact name or the full GAV.
```

and `qs add openshift` works fine.

but `qs add openshift metrics` just fails silently.

### Expected behavior
qs should report the failure not just be silent.

**Comment by aadityasinha-dotcom** [comment]
Can I work on this issue?

**Comment by maxandersen** [comment]
@aadityasinha-dotcom feel free to submit a PR.

**Comment by aadityasinha-dotcom** [comment]
Thanks for your reply. Can you briefly explain me about because I am new to open source contributions.

**Comment by maxandersen** [comment]
For creating pull request see https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request

For the issue see the description and then locate the error and try fix it :)

**Comment by ebullient** [comment]
Resolved by  #16353
